AuthorizationCodeFlow/appsettings.placeholder.json

```diff
@@ -1,11 +1,11 @@
 {
   "Resource": "https://dat-b.osisoft.com",
-  "TenantId": "REPLACE_WITH_TENANT_ID",
-  "ClientId": "REPLACE_WITH_CLIENT_ID",
+  "TenantId": "PLACEHOLDER_REPLACE_WITH_TENANT_ID",
+  "ClientId": "PLACEHOLDER_REPLACE_WITH_CLIENT_ID",
   "ApiVersion": "v1",
   "RedirectHost": "https://127.0.0.1",
   "RedirectPort": "54567",
   "RedirectPath": "signin-oidc",
-  "Username": "TESTONLY_REPLACE_WITH_USERNAME",
-  "Password": "TESTONLY_REPLACE_WITH_PASSWORD"
+  "Username": "PLACEHOLDER_REPLACE_WITH_USERNAME",
+  "Password": "PLACEHOLDER_REPLACE_WITH_PASSWORD"
```

Comment:
along the same line as Derek's comment, changing the strings here to say testonly would be useful too

Reply:
I changed the placeholder to include ```_TEST_ONLY```